### PR TITLE
fixed error

### PR DESCRIPTION
**Pull request by ronniebhatt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@groceristar/pdf-export": "^1.0.41",
```
Comment (atherdon) [inline package.json]: you installing this package into this package

**Comment by atherdon** [comment]
as you mentioned React - this pull request didn't approved. and you need to sync your fork too, because another developer made some changes as well

**Comment by atherdon** [comment]
you mentioned that this task solved, right? Can you give me a link to your task? I want to take a look. I don't think that solved your task, maybe only by coincidence...

**Comment by atherdon** [comment]
@ronniebhatt should i close this pull request?

**Comment by ronniebhatt** [comment]
Which task i should solve

**Pull request closed without merging**